cql3/multi_column_relation.hh

```diff
@@ -99,6 +101,15 @@ public:
         return create_multi_column_relation(std::move(entities), relation_type, std::move(values_or_marker), {}, {});
     }
 
+    /**
+     * Same as above, but sets the magic mode that causes us to treat the restrictions as "raw" clustering bounds
+     */
+    static shared_ptr<multi_column_relation> create_scylla_clustering_bound_non_in_relation(std::vector<shared_ptr<column_identifier::raw>> entities,
+                                                                    expr::oper_t relation_type, shared_ptr<term::multi_column_raw> values_or_marker) {
+        assert(relation_type != expr::oper_t::IN);
```

Comment:
Assert? Kill the server for this? Or is another layer ensuring it won't happen?

Reply:
I honestly just copied the routine above, renamed and modified the parameterization on object created. But fwiw, the parser absolutely should.